packages/driver/src/cy/keyboard.ts

```diff
@@ -23,17 +24,6 @@ export interface KeyboardModifiers {
   shift: boolean
 }
 
-export interface KeyboardState {
-  keyboardModifiers?: KeyboardModifiers
-}
-
-export interface ProxyState<T> {
-  <K extends keyof T>(arg: K): T[K] | undefined
-  <K extends keyof T>(arg: K, arg2: T[K] | null): void
-}
-
-export type State = ProxyState<KeyboardState>
-
```

Comment:
This code is added before `StateFunc` type is created. Now, it's OK to remove them.